src/client/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Modal.styles.css';
+import addedCart from '../../assets/images/addedCart.png';
+
+export default function ModalComponent({ show, handleLink, handleCloseModal }) {
+  if (show) {
```

Comment:
I will recommend that you use createPortal to render things outside of the normal flow, read more about it [here](https://reactjs.org/docs/portals.html#gatsby-focus-wrapper)

Reply:
thanks..i will check...